runtime/doc/options.txt

```diff
@@ -5203,9 +5203,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	%	When included, save and restore the buffer list.  If Vim is
 		started with a file name argument, the buffer list is not
 		restored.  If Vim is started without a file name argument, the
-		buffer list is restored from the shada file.  Quickfix 
-		('buftype'), unlisted ('buflisted'), unnamed and buffers on 
-		removable media (|shada-r|) are not saved.
+		buffer list is restored from the shada file. Quickfix ('buftype'), 
+		unnamed and buffers on removable media (|shada-r|) are not saved.
```

Comment:
Removed `unlisted ('buflisted')` from `shada-%`, as `nobuflisted` files are not stored in general.

Reply:
Still worth mentioning explicitly I think. Let's keep the help as it was here.

Reply:
Okay, lets do that. 

The change is reverted by: e88fd1cb512d525d83dde3a1113335e9785f777d